common/src/types.rs

```diff
@@ -32,24 +32,26 @@ impl BorshSerialize for U256 {
     where
         W: io::Write,
     {
-        BorshSerialize::serialize(&self.0 .0, writer)
+        let mut v = [0u8; 32];
+        self.0.to_big_endian(&mut v);
```

Comment:
The U256 is encoded in little endian
https://docs.rs/ethereum-types/latest/ethereum_types/struct.U256.html

> Little-endian large integer type 256-bit unsigned integer.

We are encoding it here to big endian, will this cause problems in the future?